imports/client/components/ProfileList.tsx

```diff
@@ -44,13 +48,170 @@ const ImageBlock = styled.div`
   justify-content: center;
 `;
 
+const OperatorBox = styled.div`
+  margin-left: auto;
+  padding-right: 0.5rem;
+  * {
+    margin: 0 0.25rem;
+  }
+`;
+
+type OperatorModalHandle = {
+  show(): void;
+};
+
+const PromoteOperatorModal = React.forwardRef((
+  { profile, huntId }: { profile: ProfileType, huntId: string },
+  forwardedRef: React.Ref<OperatorModalHandle>,
+) => {
+  const [visible, setVisible] = useState(true);
+  const show = useCallback(() => setVisible(true), []);
+  const hide = useCallback(() => setVisible(false), []);
+  useImperativeHandle(forwardedRef, () => ({ show }), [show]);
+
+  const demote = useCallback(() => {
+    Meteor.call('makeOperatorForHunt', profile._id, huntId);
```

Comment:
I'd prefer we waited for the successful reply from the server before dismissing the dialog, and displayed the failure message to the user on failure